File: Optional5/t2.java
public class t2{
    public static void main(String[] args){
        double[] value = {2.5, -4.75, 1.2, 3.67};
        System.out.println("The biggest number is " + exchange(value));
    }
    private static double exchange(double[] nums){
        double biggest = nums[0];
        for (int i = 0; i < nums.length; i++){
            if (nums[i] > biggest)
                biggest = nums[i];
        }
        return biggest;
    }
}

File: Optional5/t2.cpp
#include <stdio.h>
float exchange(float a[], int n);
int main(){
    float value[4] = {2.5, -4.75, 1.2, 3.67};
    exchange(value, 4);
    printf("Value[0] = %f", exchange(value, 4));
    return 0;
}
float exchange(float a[], int n){
    float biggest = a[0];
    for (int i = 1; i < 4; i++){
        if (a[i] > biggest){
            biggest = a[i];
        }
    }
    return biggest;
}